shared/libretime_shared/config/_models.py

```diff
@@ -197,6 +197,31 @@ class AudioOpus(BaseAudio):
     format: Literal[AudioFormat.OPUS] = AudioFormat.OPUS
 
 
+class HlsStream(BaseModel):
+    fragment_prefix: str = "mp3_128"
+    bitrate: str = "128k"
+    sample_rate: int = 44100
+    codec: str = "libmp3lame"
+
+
+class HlsOutput(BaseModel):
+    kind: Literal["hls"] = "hls"
+    enabled: bool = False
+    public_url: Optional[AnyUrl] = None
+    host: str = "localhost"
+    port: int = 80
```

Comment:
This can be removed, I don't see it being used anywhere

Reply:
ok